src/Containers/CollapsibleAccordion/CollapsibleAccordionItem.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState, useRef, useLayoutEffect } from 'react';
+import styled from 'styled-components';
+import { AngleRight } from '@styled-icons/fa-solid/AngleRight';
+import { ShoppingCart } from '@styled-icons/remix-line/ShoppingCart';
+import { Mixins } from '../../Utils';
+import { transition } from '../../Utils/Mixins';
+import { Heading } from '../../Text';
+
+/* Max number shown on badge */
+const ROTATE_ARROW_POINT_DOWN = 90;
+
+export interface CollapsibleAccordionItemProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    /* Heading text */
+    header: string;
+    /* Month and Year of purchases */
+    purchaseDate: string;
+    /* Total amount of purchases */
+    purchaseCount: number;
+}
+
+export interface SectionDivProps {
+    isActive: boolean;
```

Comment:
Missing comments for all typescript interfaces

Reply:
Gonna be honest... Since I inherited the code and didn't actually write it, I don't know what to put as the comments

Reply:
The original code is missing comments for its pieces btw

Reply:
I believe the is active translates to isOpened lol. But if you don't know the code how are you using it 😂. Ok you can merge but if you have free time later I would revisit this.